ydb/core/kqp/gateway/kqp_metadata_loader.cpp

```diff
@@ -892,18 +892,15 @@ NThreading::TFuture<TTableMetadataResult> TKqpTableMetadataLoader::LoadTableMeta
                     }
                     case EKind::KindIndex: {
```

Comment:
Can you please explain what this code will do for the vector index? When will it be called?

Reply:
The function loads table metadata from KQP gateway cache.
This cache is used for query compilation.